specs/shell/launch-clipboard-buffer-url-list.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: Launch tabs with clipboard URLs
+command: "pbpaste | xargs -n1 -I{} open {}"
+tags:
+  - pbpaste
+  - xargs
+  - macosx
+  - clipboard
+  - urls
+description: Launch all URLs (arranged in a list in the clipboard buffer). Will be launched using default browser and tab creation preference. Useful for copying a column of URLs from a spreadsheet and visiting them all at once.
+author: DC
```

Comment:
could you use your github username and URL? If you're not comfortable with that also feel free to remove both fields entirely